app/src/main/res/values-ar-rSA/strings.xml

```diff
@@ -59,5 +59,6 @@
 
     The first element is the day, and the second element is the time
     -->
+    <string name="time_template">%1$s في %2$s</string>
```

Comment:
`time_template` under `ar-rSA` uses a RTL language + string formatter. I'm not sure if the produced outcome is correct, but I haven't been able to fix it. See https://developer.android.com/training/basics/supporting-devices/languages#FormatTextExplanationSolution for more info